cypress/support/e2e.ts

```diff
@@ -6,125 +6,17 @@
-            win.localStorage.setItem(
-              'redux_localstorage_simple_user', // storage key for the user reducer using 'redux-localstorage-simple'
-              JSON.stringify(options?.userState ?? CONNECTED_WALLET_USER_STATE)
-            )
-
-            // Set feature flags, if configured.
-            if (options?.featureFlags) {
-              const featureFlags = options.featureFlags.reduce((flags, flag) => ({ ...flags, [flag]: 'enabled' }), {})
-              win.localStorage.setItem('featureFlags', JSON.stringify(featureFlags))
-            }
-
-            // Inject the mock ethereum provider.
-            if (options?.ethereum === 'hardhat') {
-              win.ethereum = provider
-            } else {
-              win.ethereum = injected
-            }
-          },
-        })
-      )
-  }
-)
-
-beforeEach(() => {
-  cy
-    // Many API calls enforce that requests come from our app, so we must mock Origin and Referer.
-    .intercept('*', (req) => {
-      req.headers['referer'] = 'https://app.uniswap.org'
-      req.headers['origin'] = 'https://app.uniswap.org'
-    })
-    // Infura uses a test endpoint, which allow-lists http://localhost:3000 instead.
-    .intercept(/infura.io/, (req) => {
-      req.headers['referer'] = 'http://localhost:3000'
-      req.headers['origin'] = 'http://localhost:3000'
-      req.alias = req.body.method
-      req.continue()
-    })
-    // Mock Amplitude responses to avoid analytics from tests.
-    .intercept('https://api.uniswap.org/v1/amplitude-proxy', (req) => {
-      const requestBody = JSON.stringify(req.body)
-      const byteSize = new Blob([requestBody]).size
-      req.reply(
-        JSON.stringify({
-          code: 200,
-          server_upload_time: Date.now(),
-          payload_size_bytes: byteSize,
-          events_ingested: req.body.events.length,
-        })
-      )
-    })
-    // Mock our own token list responses to avoid the latency of IPFS.
-    .intercept('https://gateway.ipfs.io/ipns/tokens.uniswap.org', TokenListJSON)
-    .intercept('https://gateway.ipfs.io/ipns/extendedtokens.uniswap.org', { statusCode: 201, body: { tokens: [] } })
-    .intercept('https://gateway.ipfs.io/ipns/unsupportedtokens.uniswap.org', { statusCode: 201, body: { tokens: [] } })
-    // Reset hardhat between tests to ensure isolation.
-    // This resets the fork, as well as options like automine.
-    .hardhat()
-    .then((hardhat) => hardhat.reset())
-})
+import './commands'
+import './setupTests'
+
+// Squelch logs from fetches, as they clutter the logs so much as to make them unusable.
+// See https://docs.cypress.io/api/commands/intercept#Disabling-logs-for-a-request.
+// TODO(https://github.com/cypress-io/cypress/issues/26069): Squelch only wildcard logs once Cypress allows it.
+const log = Cypress.log
+Cypress.log = function (options, ...args) {
+  if (options.displayName === 'script' || options.name === 'request') return
+  return log(options, ...args)
+} as typeof log
```

Comment:
will this make it difficult to debug issues related to failing requests? I agree the volume is not manageable currently, but my first thought is that a more selective approach would be better.

Reply:
Yes, it will. I can update this in a follow-up to be more selective.